src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -4,11 +4,11 @@
 #include "pch.h"
 #include "OpenTerminalHere.h"
 #include "../WinRTUtils/inc/WtExeUtils.h"
+#include "../WinRTUtils/inc/LibraryResources.h"

```

Comment:
this whole project is pretty much one and a half .cpp files. Is there value in having this in the PCH? Maybe.